storage/qcs.go

```diff
@@ -14,6 +14,9 @@ type QuorumCertificates interface {
 	// StoreTx stores a Quorum Certificate as part of database transaction QC is indexed by QC.BlockID.
 	// * storage.ErrAlreadyExists if any QC for blockID is already stored
 	StoreTx(qc *flow.QuorumCertificate) func(*transaction.Tx) error
+
+	// * storage.ErrAlreadyExists if any QC for blockID is already stored
+	StorePebble(qc *flow.QuorumCertificate) func(PebbleReaderBatchWriter) error
```

Comment:
It's hard to unify the Store function for both badger and pebble. Because badger needs to support transaction and pebble needs to support batch update. And the API for transaction and batch update are different. 

So I ended up create a new method `StorePebble`. badger will not implement `StorePebble`, and pebble will not implement `StoreTx`.